tools/db_crashtest.py

```diff
@@ -124,8 +124,9 @@ def is_direct_io_supported(dbname):
     "max_background_compactions": 1,
     "max_bytes_for_level_base": 67108864,
     "memtablerep": "skip_list",
-    "prefixpercent": 25,
-    "readpercent": 25,
+    "prefixpercent": 0,
+    "readpercent": 50,
+    "prefix_size" : -1,
```

Comment:
Why do we need two parameters to enable/disable prefix tests?

Reply:
prefixpercent is to determine whether to run the queries about prefix scan, and "prefix_size" is to determine prefix_extractor to use. Unless we disable prefix extractor, other tests will still go through it.